release_notes/374-release-notes.adoc

```diff
@@ -379,6 +380,16 @@ Release date: 2 May 2023
 * This release of {product-title-short} includes a fix for link:https://access.redhat.com/security/cve/CVE-2023-28617[CVE-2023-28617] for {op-system-base}.
 * This release includes a fix for an issue with migration to the PostgreSQL database (Technology Preview).
 
+[id="resolved-in-version-3744"]
+=== Resolved in version 3.74.4
+
+Release date: 31 May 2023
+
+* {product-title-short} 3.74.4 is built with updated golang to fix the following CVEs:
+** link:https://access.redhat.com/security/cve/cve-2023-24540[CVE-2023-24540]
+** link:https://nvd.nist.gov/vuln/detail/cve-2023-24539[CVE-2023-24539]
+** link:https://nvd.nist.gov/vuln/detail/cve-2023-29400[CVE-2023-29400]
```

Comment:
s/ Maybe you could write a one-liner about "what is being fixed" 
eg: CVE-2023-24540: Fixed .......
Same for the other two, it is up to you though!

Reply:
Thank you. That's what we have in Errata and other release notes, so I'm keeping it as it is.